kbi/0008/index.rst

```diff
@@ -0,0 +1,106 @@
+.. index::
+   triple: git-annex; uninit; remove dataset annex
+
+KBI0008: Remove a dataset's annex
+=================================
+
+:authors: Christian Mönch <christian.moench@web.de>
+:discussion: https://github.com/datalad-handbook/book/issues/939
+:keywords: annex, remove annex, annex uninit
+
+This knowledge base item describes how to remove the annex from a DataLad
+dataset and propagate that change to siblings
+
+DataLad version used: this knowledge base item was created based on DataLad version 0.18.3
+
+
+The scenario
+------------
+
+You have a DataLad dataset that has annexed files and you decide that you
+don't want to use git-annex anymore in this dataset. This change should
+also be propagated to all siblings of the dataset.
+
+
+A few words of caution
+......................
+
+1. You should not remove the annex if you have large files, i.e. 1GB or larger, in your dataset. ``git`` is not intended for large files, which is the reason for the creation of ``git annex`` in the first place.
+
+2. If you remove the annex, your files will be untracked in your dataset directory. That means, they are not versioned at this moment. If you don't follow the instructions in this knowledge base item precisely you might lose those files. Please be careful when you remove the annex! Work on a copy, or make sure that you have a sibling that contains your data. We do not want you to lose data.
```

Comment:
```suggestion
2. Once you removed an annex, previously annexed files will then be untracked in the dataset's directory. That means, they are no longer versioned at that moment. If you do not follow these instructions precisely **you might lose the files' content**. It is safer to work on a dataset copy, or make sure that another dataset sibling still contains all files' content.
```

Style change suggestion. Focus less on intentions and personal connection, and more on facts.